docs/project/list-of-diagnostics.md

```diff
@@ -1,15 +1,52 @@
-List of Obsoletions
-==================
+# List of Diagnostics Produced by .NET Libraries APIs
 
-Per https://github.com/dotnet/designs/blob/master/accepted/2020/better-obsoletion/better-obsoletion.md, we now have a strategy in place for marking existing APIs as `[Obsolete]`. This takes advantage of the new diagnostic id and URL template mechanisms introduced to `ObsoleteAttribute` in .NET 5.
+## Obsoletions
 
-When obsoleting an API, use the diagnostic ID `SYSLIB####`, where _\#\#\#\#_ is the next four-digit identifier in the sequence, and add it to the list below. This helps us maintain a centralized location of all APIs that were obsoleted using this mechanism.
+Per https://github.com/dotnet/designs/blob/master/accepted/2020/better-obsoletion/better-obsoletion.md, we now have a strategy for marking existing APIs as `[Obsolete]`. This takes advantage of the new diagnostic id and URL template mechanisms introduced to `ObsoleteAttribute` in .NET 5.
 
-The URL template we use for obsoletions is `https://aka.ms/dotnet-warnings/{0}`.
+The diagnostic id values reserved for obsoletions are `SYSLIB0001` through `SYSLIB0999`. When obsoleting an API, claim the next three-digit identifier in the `SYSLIB0###` sequence and add it to the list below. The URL template for all obsoletions is `https://aka.ms/dotnet-warnings/{0}`. The `{0}` placeholder is replaced by the compiler with the `SYSLIB0###` identifier.
```

Comment:
> values reserved for obsoletions are `SYSLIB0001` through `SYSLIB0999`

Can you add a comment to https://github.com/dotnet/runtime/blob/main/src/libraries/Common/src/System/Obsoletions.cs about the expected range?

Reply:
Good idea. I added a comment block in that file both pointing to this doc and showing the expected range.